### DISCUSSION TITLE
GSA Mod Function Request - DinoColoring Brush Use - Logging

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#579] [Suggestions]
I'd like to request a logging feature in GSA's paint brush tool, to log the user ID, or character name + tribe name, of players that use up a dino paint brush. This function would be beneficial for admins corroborating claims of paint brushes being consumed or lost without merely having to go off of trust or word-of-mouth, inevitably improving user experience. 

Currently, logs are thus: 
```
11/24/2023, 9:22:58 [GSA] 0002a2a736ce4e459612766ed447a91d Got His Brush
11/24/2023, 9:23:48 [GSA] DinoColoring succeeded!
```

They could be improved with: 
```
11/24/2023, 9:22:58 [GSA] (Character: NR (0002a2a736ce4e459612766ed447a91d)) of (Tribe: Headhunters (1352203106)) got 1 DinoColoring Brush.
11/24/2023, 9:23:48 [GSA] DinoColoring succeeded! Brush used by (Character: NR (0002a2a736ce4e459612766ed447a91d)) of (Tribe: Headhunters (1352203106)) on (Dino: Spinosaurus (Typhoon) - Lvl 240 (290945538258663227))! 
```

**Comment by mvaessen** [comment]
the mod should have some extra logging, but not as much as suggested. It will report the character ID and the dino ID.
